charts/k8tz/README.md

```diff
@@ -48,36 +48,41 @@ helm upgrade --install k8tz k8tz/k8tz \
+| service.type                       | Admission controller service type                                                                                                                                             | ClusterIP         |
+| service.port                       | Admission controller service port                                                                                                                                             | 443               |
+| resources                          | Resource requests and limitations for the admission controller deployment                                                                                                     | {}                |
+| nodeSelector                       | Node selector for the admission controller deployment                                                                                                                         | {}                |
+| tolerations                        | Tolerations for the admission controller deployment                                                                                                                           | {}                |
+| affinity                           | Affinities and anti-affinities for the admission controller deployment                                                                                                        | {}                |
+| webhook.failurePolicy              | Failure policy for the admission webhook. May be `Fail` or `Ignore`                                                                                                           | `Fail`            |
+| webhook.certManager.enabled        | Use `cert-manager` to manage the webhook certificate by using `Certificate` resource                                                                                          | false             |
+| webhook.certManager.secretTemplate | Add custom labels and annotations to `Secret` that containing certificate generated by cert-manager. Please refer to cert-manager documentation for using this                | {}                |
+| webhook.certManager.duration       | The duration for the `Not After` date for the certificate generated by cert-manager. Please refer to cert-manager documentation for using this                                | 5114d             |
+| webhook.certManager.renewBefore    | How long before the currently issued certificate’s expiry cert-manager should renew the certificate. Please refer to cert-manager documentation for using this                | {}                |
+| webhook.certManager.issuerRef      | Reference to cert-manager issuer deployed on the cluster. Please refer to cert-manager documentation for using this                                                           | name: selfsigned <br>kind: Issuer |
```

Comment:
the correct default is: `{ "kind": "Issuer", "name": "selfsigned" }`.
and I am wondering if its a good default since installing k8tz creating the namespace `k8tz`, so it make no sense that there will be `Issuer` (namespaced object) there prior to installation. maybe the default kind should be `ClusterIssuer`?

Reply:
for the default value, I prefer using the two different lines style

how about adding our own `Issuer` resource within k8tz namespace using `-k8tz` as suffix? ex: `selfsigned-k8tz`. is this too far from the scope?

Reply:
> for the default value, I prefer using the two different lines style
> 
> how about adding our own `Issuer` resource within k8tz namespace using `-k8tz` as suffix? ex: `selfsigned-k8tz`. is this too far from the scope? 
> 

I think this it is a bit out of scope. Just change the defualt kind to ClusterIssuer to hint the user the the issuer should be non namespaced.